.gitignore

```diff
@@ -1,4 +1,5 @@
 .php_cs.cache
+.DS_Store
```

Comment:
Eigentlich gehören für mich solche Betriebssystem-abhängigen (oder auch IDE-abhängigen) Dateien nicht in die Projekt-gitignore, sondern die sollte derjenige, der das OS (oder die IDE) nutzt, bei sich global ignorieren.
Man kann für git eine globale gitignore anlegen.

Ist mir aber auch nicht total wichtig, kann also wegen mir auch doch rein. Bisher haben wir solche Dateien aber immer weggelassen.

Reply:
Ich nehme es gerne wieder raus. Anders wäre mir auf dem Fremdrechner dieser PR nicht möglich gewesen.